app/components/new-programyear.js

```diff
@@ -4,6 +4,7 @@ import { action } from '@ember/object';
 import { dropTask } from 'ember-concurrency';
 
 export default class NewProgramyearComponent extends Component {
+  @tracked allYears;
```

Comment:
Doesn't matter in any way, but since this value never changes it doesn't actually need to be `@tracked`.

Reply:
:heavy_check_mark: